Pyxides/im/argo.py

```diff
@@ -16,7 +16,7 @@
 from Pyxis.ModSupport import *
 
 # register ourselves with Pyxis and define the superglobals
-register_pyxis_module(superglobals="MS LSM")
+register_pyxis_module()
```

Comment:
I got this error when I import `im.argo` when MS is not set. 
```
/home/makhathini/devel/simms/pyxis/Pyxis/Internals.pyc in assign_templates()
    665     if ns is not Pyxis.Context:
    666       for var in _superglobals.get(id(ns)):
--> 667         ns[var] = Pyxis.Context[var];
    668   # set logger, in case LOG value has changed
    669   set_logfile(Pyxis.Context.get('LOG',None));

KeyError: 'MS'
```
How can I avoid this?